provider/filestore.go

```diff
@@ -103,7 +103,8 @@ func (store *AzureFileStore) containerName() string {
 	return store.ContainerName
 }
 
-func (store *AzureFileStore) addAzureVars(envVars map[string]string) map[string]string {
+func (store AzureFileStore) AddEnvVars(envVars map[string]string) map[string]string {
```

Comment:
golang convention is to use either value or pointer receivers on a struct (not both -- due to confusion). Can we do that here?

Reply:
Was this resolved @ahmadnazeri